test/sql/include/ddl_ops_1.sql

```diff
@@ -88,3 +88,61 @@ ALTER TABLE my_ht DROP COLUMN IF EXISTS val3;
 -- Should skip and not error
 ALTER TABLE my_ht DROP COLUMN IF EXISTS val3;
 \d my_ht
+
+--default indexes--
+--both created
+BEGIN;
+CREATE TABLE PUBLIC."Hypertable_1_default_index" (
```

Comment:
Nit. Confusing table name; sounds like this is an index. Makes it harder to understand test output. Maybe use `Hypertable_1_with_default_index_enabled` or similar?